versioned_docs/version-8.1/components/connectors/out-of-the-box-connectors/aws-sns-inbound.md

```diff
@@ -0,0 +1,156 @@
+---
+id: aws-sns-inbound
+title: Amazon Simple Notification Inbound Service Connector
+sidebar_label: Amazon SNS Inbound Connector
+description: Subscribe to your AWS SNS topic notification
+---
+
+The **Amazon Simple Notification Service (SNS) Inbound Connector** is a Connector that allows you to start or continue
+a BPMN process triggered by an [AWS SNS](https://console.aws.amazon.com/sns/home) notification.
+
+## Create an AWS SNS Inbound Connector task
+
+1. Start building your BPMN diagram. You can use the **AWS SNS Inbound Connector** with either **Start Event** or **Intermediate Catch Event** building blocks.
+2. Select the applicable element and change its template to an **AWS SNS Inbound Connector**.
+3. Fill in all required properties.
+4. Complete your BPMN diagram.
+5. Deploy the diagram to activate the webhook.
+6. Navigate to the **Webhooks** tab in the properties panel to see the webhook URL.
+
+## Make your AWS SNS Inbound Connector for receiving notifications executable
+
+![Start event SNS connector](../img/connectors-aws-sns-inbound-start-filled.png)
+
+![Intermediate catch event SNS connector](../img/connectors-aws-sns-inbound-intermediate-filled.png)
```

Comment:
can you add a subtitle, that one is SNS as start and the other one that SNS as intermediate event.
That would make it easier to read

Reply:
Done